sylvia-derive/src/message.rs

```diff
@@ -916,15 +917,52 @@ impl EntryPoints {
             )
             .unwrap_or_else(|| parse_quote! { #sylvia ::cw_std::StdError });
 
-        Self { name, error }
+        let reply = source
+            .items
+            .iter()
+            .filter_map(|item| match item {
+                ImplItem::Method(method) => Some(method),
+                _ => None,
+            })
+            .find(|method| {
+                method.attrs.iter().any(|attr| {
+                    if !attr.path.is_ident("msg") {
+                        return false;
+                    }
+
+                    let attr = match MsgAttr::parse.parse2(attr.tokens.clone()) {
+                        Ok(attr) => attr,
+                        Err(_) => return false,
+                    };
+
+                    attr == MsgType::Reply
+                })
+            })
+            .map(|method| method.sig.ident.clone());
```

Comment:
MsgVariants::new(source.item()).0.find(|variant| variant.msg_ty == MsgType::Reply).map(|method| &method.sig.ident);

Reply:
Oh, I was really looking for that thing, thanks